src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+// STATE
+const initialCategoriesState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+// CREATE A SLICE USING createSlice
+const categoriesSlice = createSlice({
+  name: 'category',
+  initialState: initialCategoriesState,
+  reducers: {
+    updateStatus: (state) => {
+      state.status = 'Under construction';
+    },
+  },
+});
+
+export const { status } = categoriesSlice.actions;
```

Comment:
- [ ] The action you are exporting here says `status` however there is no such action created by you, what you currently have is `updateStatus` as such this should be the action that should be exported :+1: :sparkles: